scripts/csmith/install-csmith.sh

```diff
@@ -0,0 +1,6 @@
+git clone https://github.com/csmith-project/csmith.git
```

Comment:
Is the csmith testing script complete enough to "publish" on REBAR? Is it useful to have in REBAR, instead of moving this inside the boom src code directory?

Reply:
I don't know if anyone else will use `csmith` so it might be good to just move it to BOOM